cdk_infra/lib/app.ts

```diff
@@ -10,6 +10,12 @@ const envDefault = {
   account: process.env.CDK_DEFAULT_ACCOUNT,
   region: process.env.CDK_DEFAULT_REGION
 };
+// Lambda CI runs in us-east-1
+const lambdaDefaults = {
+  account: process.env.CDK_DEFAULT_ACCOUNT,
+  region: 'us-east-1'
```

Comment:
Is the deployment workflow prepared for multi region deployments? Will there be any issue with the `AWS_REGION` env var being set?